packages/terra-button/src/Button.scss

```diff
@@ -28,24 +24,24 @@
   // stylelint-enable selector-no-qualifying-type
 
   .button {
-    border-radius: var(--terra-button-border-radius, 0.25em);
+    border-radius: var(--terra-button-border-radius, 0.2142rem);
     border-style: solid;
-    border-width: var(--terra-button-border-width, 1px);
+    border-width: 1px;
     cursor: pointer;
     display: inline-block;
-    font-size: map-get($terra-button-font-sizes, medium);
-    font-weight: var(--terra-button-font-weight, 400);
-    line-height: 1.429; // 20px when html font size is computed to 14px
-    padding-bottom: var(--terra-button-padding-bottom, 0.286em);
-    padding-left: var(--terra-button-padding-left, 0.714em);
-    padding-right: var(--terra-button-padding-right, 0.714em);
-    padding-top: var(--terra-button-padding-top, 0.286em);
+    font-family: inherit;
+    font-size: inherit;
+    line-height: var(--terra-button-line-height, 1.429);
+    outline: none;
+    padding-bottom: var(--terra-button-vertical-padding, 0.4285rem);
```

Comment:
update 0.4285rem to `0.286rem` to match 4px padding-top (probably Zeplin error)

Reply:
I think I missed it while simplifying the theming vars :)